services_app/src/main/res/layout/fragment_main.xml

```diff
@@ -144,13 +144,14 @@
         android:id="@+id/btnSignInMain"
         style="@style/ButtonStyle"
         android:layout_width="0dp"
-        android:layout_height="wrap_content"
+        android:layout_height="50dp"
         android:layout_margin="@dimen/portrait_regular_margin"
         android:text="@string/drawer_sign_in_button_text"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/tvLastSyncTimeMain"
+
         tools:ignore="TextContrastCheck" />
 
 </androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
This is a very small issue. GitHub does not like a file that does not end in a blank line. So if it doesn't end in a blank line a red circle with a dash in it will appear point it out.

Reply:
Oh had no idea github have problems with blank lines. I'll clean that up in the next PR